components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1103 @@
+$$ (discrete-mom-del4)
+
+### Velocity dissipation across a sloping $\tilde{z}$ surface
+
+We interpret $\left<\mathbf{u}^\prime \tilde{u}^\prime \right>$ as the dissipation of momentum across the sloping $\tilde{z}$ surface.
+
+$$
+\left<\mathbf{u}^\prime \tilde{u}^\prime \right> = \left\{\left[\nu_{2,e} \left( \nabla \tilde{D}_{i} - \nabla^{\perp} \tilde{\zeta}_{v} \right)\right]_k - \left[\nu_{4,e} \nabla^2 \left( \nabla^2 \tilde{u}_{e,k} \right)\right]_k\right\}
+$$ (discrete-mom-flux-sloping)
+
+While it looks very similar to [](#discrete-mom-del2) - [](#discrete-mom-del4), there are a few critical differences.  First, the normal velocities in the divergence and relative vorticity in [](#discrete-mom-flux-sloping) are the reconstructed velocity at the top of the cell along an edge, not the layer average.  Second, the velocities in the divergence and relative vorticity are only the projection across the interface (hence the tilde on $D$ and $\zeta$), computed in a discrete sense following
+
+$$
+\left[\tilde{u}_{e}\right]_k = u_e \nabla \tilde{z}_{e,k}
+$$
+
+in this relation, we have moved the subscript $k$ off the variable itself to prevent confusion with the layer average.  With this definition, [](#discrete-mom-flux-sloping) goes to zero for flat layer surfaces.
+
+#### Vertical velocity dissipation
+The vertical turbulent momentum stress is most commonly parameterized as a down-gradient process, i.e.,
+
+$$
+\left[ \left<\mathbf{u}^\prime \tilde{w}_{tr}^\prime \right> \right]_{e,k} = -\frac{\nu_v \rho}{\rho_0} \left[\frac{\partial u}{\partial \tilde{z}}\right]_{e,k}
+$$
+
+Plugging this relation into the last part of [](#discrete-momentum)
+
+$$
+\frac{1}{\left[\tilde{h}_{i,k}\right]_{e,k}}  \left\{ \left[ \nu_v \left[\frac{\partial u}{\partial \tilde{z}}\right]_{e,k} \right]_{e,k} - \left[ \nu_v \left[\frac{\partial u}{\partial \tilde{z}}\right]_{e,k} \right]_{e,k+1}  \right\}
+$$ (discrete-mom-vert-diff)
+
+##### Vertical derivatives in a finte volume framework
+
+Since, Omega will predict layer average quantities (e.g., $u_k$), it's not immediately clear that a traditional discretization is appropriate as there is a discontinuity in the predicted variables at the layer edge.  To circumvent this problem, we turn to weak derivatives instead of traditional pointwise forms.  A weak derivative of $\varphi$ is defined as
+
+$$
+\int_{\tilde{z}_{k+1/2}^{\text{top}}}^{\tilde{z}_{k-1/2}^{\text{top}}} \varphi^\prime \psi d\tilde{z} = \varphi \psi - \int_{\tilde{z}_{k+1/2}^{\text{top}}}^{\tilde{z}_{k-1/2}^{\text{top}}} \varphi \psi^\prime d\tilde{z}.
+$$ (weak-derivative-defn)
+
+Here $\psi$ is a smooth test function.  If we choose $\psi$ such that it has compact support on the interval $(\tilde{z}_{k+1/2}^{\text{top}},\tilde{z}_{k-1/2}^{\text{top}})$ then the first term on the right hand side of [](#weak-derivative-defn) is zero.  Using the fact that Omega predicts layer averages, the integral on the right hand side can be broken into
```

Comment:
```suggestion
Here $\psi$ is a smooth test function.  If we choose $\psi$ such that it has compact support on the interval $]\tilde{z}_{k+1/2}^{\text{top}},\tilde{z}_{k-1/2}^{\text{top}}[$ then the first term on the right hand side of [](#weak-derivative-defn) is zero.  Using the fact that Omega predicts layer averages, the integral on the right hand side can be broken into
```
then we have clearly set ourselves up for zero values on end points.

Reply:
I don't think that is right.  In my experience () in a range means the end points are not included.  I don't understand the backwards bracket

Reply:
ok, then it is just a different notation convention. Let's keep it as is.